packages/transport/test/end2end/End2End.test.ts

```diff
@@ -420,6 +420,51 @@ describe("RelationshipTest: Revoke", function () {
     });
 });
 
+describe("RelationshipTest: Terminate", function () {
+    let connection: IDatabaseConnection;
+    let transport: Transport;
+
+    let from: AccountController;
+    let to: AccountController;
+
+    beforeAll(async function () {
+        connection = await TestUtil.createDatabaseConnection();
+        transport = TestUtil.createTransport(connection);
+
+        await transport.init();
+
+        const accounts = await TestUtil.provideAccounts(transport, 2);
+        from = accounts[0];
```

Comment:
Perhaps the names can be chosen better, such if when you read them, you know they describe accounts.

Reply:
you essentially propose refactoring the transport library :)